aws/logs_monitoring/lambda_function.py

```diff
@@ -74,12 +74,33 @@ def __init__(self, name, pattern, placeholder):
         "xxxxx@xxxxx.com",
     ),
     ScrubbingRuleConfig(
-        "CUSTOM_SCRUBBING_RULE", 
-        os.getenv("CUSTOM_SCRUBBING_RULE", default=None), 
-        os.getenv("CUSTOM_SCRUBBING_RULE_REPLACEMENT", default="xxxxx")
+        "DD_SCRUBBING_RULE", 
```

Comment:
The change on scrubbing rules could have been done in another PR to ease review

Reply:
Good point!